components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/manager/FxaAccountManager.kt

```diff
@@ -753,6 +747,21 @@ open class FxaAccountManager(
         return null
     }
 
+    private suspend fun postAuthenticated(newAccount: Boolean) {
+        // Before any sync workers have a chance to access it, make sure our SyncAuthInfo cache is hot.
+        maybeUpdateSyncAuthInfoCache()
+
+        // Notify our internal (sync) and external (app logic) observers.
+        notifyObservers { onAuthenticated(account, newAccount) }
+
+        // If device supports SEND_TAB...
+        if (deviceConfig.capabilities.contains(DeviceCapability.SEND_TAB)) {
```

Comment:
I'm wonder if there are event categories that we would want to consider when to refresh and poll for. I see why we want to treat SEND_TAB in a special way, but I guess I'm curious about what happens tomorrow when we those new events coming in? Would it make sense then to have a sort of whitelist of event types that we would want check for?

Reply:
Possibly! I think for now, given how things are, polling based on capabilities makes sense. Our public API is that applications declare which capabilities they have, not which events they support (which, in retrospect, sounds like a pretty good way to model this as well!).
Once we start expanding "capabilities" list, we can also expand these checks.

Reply:
If capabilities are similar to a super-set of event types, then that could work too!

Reply:
That's how I've been thinking about them _for now_, but I'm not sure it'll hold over time. We'll see, anyway.